### ISSUE TITLE
Login Page With Chrome Not Working!

### ISSUE THREAD

**Issue by InfamyHacking** [owncloud/core#26268]
### Steps to reproduce
1. Latest version of chrome - own cloud 9.0.3
2. Go to login page - nothing happens
3.

### Expected behaviour
Tell us what should happen
I should get a response / login
### Actual behaviour
Tell us what happens instead
Nothing
### Server configuration
**Operating system**:
Ubuntu 14.04
**Web server:**
apache - this is the domain to test - cloud.infamyhacking.com
**Database:**
mysql  Ver 14.14 Distrib 5.5.49, for debian-linux-gnu (x86_64)
**PHP version:**
PHP 5.5.9-1ubuntu4.17
**ownCloud version:** (see ownCloud admin page)
9.0.3
**Updated from an older ownCloud or fresh install:**
Fresh Install 
**Where did you install ownCloud from:**
Digital Ocean One Click Application 
**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```No errors have been found.

**List of activated apps:**
Enabled:
  - activity: 2.2.1
  - comments: 0.2
  - dav: 0.1.6
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
Disabled:
  - encryption
  - external
  - files_antivirus
  - files_external
  - user_external
  - user_ldap

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**
{
    "system": {
        "updatechecker": false,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.infamyhacking.com"
        ],
        "datadirectory": "\/var\/www\/owncloud\/data",
        "overwrite.cli.url": "http:\/\/localhost",
        "dbtype": "mysql",
        "version": "9.0.3.2",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "cid",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "instanceid": "ocze73yfm4h3",
        "mail_from_address": "admin",
        "mail_smtpmode": "php",
        "mail_domain": "infamyhacking.com"
    }
}

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No
**Are you using encryption:** yes/no
No
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No

With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**
Chrome  Version 53.0.2785.143 m (64-bit)
**Operating system:**
Windows 7 Professional

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by RealRancor** [comment]
> own cloud 9.0.3

From the issue template you have filled out:

> Before reporting any issues please make sure that you're using the latest available version for your major branch (e.g. 9.0.x), see https://owncloud.org/changelog/

This avoids that you're reporting issues which might be already fixed in a newer available version (in your case 9.0.5).

**Comment by DeepDiver1975** [comment]
please reopen if this issue still exists after updating ...
